README.md

```diff
@@ -9,7 +9,7 @@ Open source series of workshops delivered by Gravitational services team.
 * [Kubernetes custom resources](crd/crd.md)
 * [Gravity 101](gravity101.md)
 * [Gravity fire drill exercises](firedrills.md)
-* [Gravity logging](logging.md)
+* [Gravity logging](logging-6.x.md)
```

Comment:
Should we have to also show up the split difference in the index?

Something like:
```md
* [Gravity logging (Gravity 5.5 and earlier)](logging-5.x.md)
* [Gravity logging (Gravity 6.0 and later)](logging-6.x.md)
```